<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Arena</class>
 <widget class="QWidget" name="Arena">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arena</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>220</y>
     <width>211</width>
     <height>139</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" rowspan="2">
     <widget class="QPushButton" name="pushButton_4">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0,0);
border: 2px solid #222;
color: rgb(105, 105, 105);</string>
      </property>
      <property name="text">
       <string>WEST</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_10">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>16</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 255, 0);
background-color: rgb(0, 0, 0);
border: 2px solid #222;</string>
      </property>
      <property name="text">
       <string>REST</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0,0);
border: 2px solid #222;
color: rgb(105, 105, 105);</string>
      </property>
      <property name="text">
       <string>NE</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_9">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0,0);
border: 2px solid #222;
color: rgb(105, 105, 105);</string>
      </property>
      <property name="text">
       <string>SE</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_8">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0,0);
border: 2px solid #222;
color: rgb(105, 105, 105);</string>
      </property>
      <property name="text">
       <string>SOUTH</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_7">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0,0);
border: 2px solid #222;
color: rgb(105, 105, 105);</string>
      </property>
      <property name="text">
       <string>SW</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0,0);
border: 2px solid #222;
color: rgb(105, 105, 105);</string>
      </property>
      <property name="text">
       <string>NORTH</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_5">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>16</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 255, 0);
background-color: rgb(0, 0, 0);
border: 2px solid #222;
</string>
      </property>
      <property name="text">
       <string>DIG</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_1">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0,0);
border: 2px solid #222;
color: rgb(105, 105, 105);
</string>
      </property>
      <property name="text">
       <string>NW</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2" rowspan="2">
     <widget class="QPushButton" name="pushButton_6">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0,0);
border: 2px solid #222;
color: rgb(105, 105, 105);</string>
      </property>
      <property name="text">
       <string>EAST</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0,0);
font: 16pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid #222;
color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>DAY</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>60</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0,0);
font: 16pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid #222;
color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>HP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0,0);
font: 16pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid #222;
color: rgb(105, 105, 105);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>60</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0,0);
font: 16pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid #222;color: rgb(105, 105, 105);</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>370</y>
     <width>211</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0,0);
border: 2px solid #222;
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(105, 105, 105);</string>
   </property>
   <property name="text">
    <string>You wake up in the woods...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>100</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0,0);
font: 16pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid #222;
color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>MP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0,0);
font: 16pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid #222;color: rgb(105, 105, 105);</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0,0);
font: 16pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid #222;color: rgb(105, 105, 105);</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>140</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0,0);
font: 16pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid #222;
color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>XP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0,0);
font: 16pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid #222;color: rgb(105, 105, 105);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>180</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0,0);
font: 16pt &quot;MS Shell Dlg 2&quot;;
border: 2px solid #222;
color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>GP</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>801</width>
     <height>611</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(25,25, 25);
color: rgb(85, 255, 0);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
border: 2px solid #222;
color: rgb(85, 255, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:24pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(105, 105, 105);
border: 2px solid #222;
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Your name</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>501</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>72</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 0);
border: 4px solid #222;
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Darkfield</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>501</width>
      <height>91</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>16</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 0);
background-color: rgb(0, 0, 0);
border: 2px solid #222;</string>
    </property>
    <property name="text">
     <string>PLAY</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>201</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(105, 105, 105);
border: 4px solid #222;
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>v.1.00</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>310</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(105, 105, 105);
background-color: rgb(0, 0, 0);
border: 2px solid #222;</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>A game about digging and killing...</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_11</sender>
   <signal>clicked()</signal>
   <receiver>groupBox</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>538</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>turnSlot()</slot>
 </slots>
</ui>
